src/components/shared/ActionBanner/index.scss

```diff
@@ -0,0 +1,39 @@
+.action-banner {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-start;
+
+  &.usa-alert {
+      background-color: #f4e3db;
+
+    &::before {
+      width: 0;
+    }
+  }
+
+  &__icon {
+    display: flex-item;
+      margin-right: 13px;
+  }
+
+  &__button {
+    float: right;
+      display: flex-item;
+      align-self: center;
+  }
+
+  &__text {
+    display: flex-item;
+      flex-grow: 2;
+
+      h2 {
+        margin-block-start: 0;
```

Comment:
What do these `margin-block-start` rules get us?

Does it do anything different than `margin: 0` ? If `margin: 0` works, it's an easier and more common CSS rule to understand.